spinn_front_end_common/utility_models/reverse_ip_tag_multi_cast_source.py

```diff
@@ -146,6 +148,17 @@ def __init__(
         # Keep the vertices for resuming runs
         self._machine_vertices = list()
 
+    def _validate_send_buffer_times(self, n_keys, send_buffer_times):
+        if send_buffer_times is None:
+            return
```

Comment:
Though technically correct, it should may be return None to be explicit.

Reply:
fixed